certbot-apache/certbot_apache/configurator.py

```diff
@@ -884,65 +921,77 @@ def _copy_create_ssl_vhost_skeleton(self, avail_fp, ssl_fp):
 
         try:
             with open(avail_fp, "r") as orig_file:
-                with open(ssl_fp, "w") as new_file:
-                    new_file.write("<IfModule mod_ssl.c>\n")
-
-                    comment = ("# Some rewrite rules in this file were "
-                              "disabled on your HTTPS site,\n"
-                              "# because they have the potential to create "
-                              "redirection loops.\n")
+                orig_file_list = [line for line in orig_file]
+                if vhost_num != -1:
+                    orig_file_list = self._create_block_segments(orig_file_list, vhost_num)
 
-                    for line in orig_file:
-                        A = line.lstrip().startswith("RewriteCond")
-                        B = line.lstrip().startswith("RewriteRule")
-
-                        if not (A or B):
-                            new_file.write(line)
-                            continue
+            if ssl_fp in self._skeletons:
+                bit = "a"
+                self._skeletons[ssl_fp].append(avail_fp)
+            else:
+                bit = "w"
+                self._skeletons[ssl_fp] = [avail_fp]
+
+            with open(ssl_fp, bit) as new_file:
```

Comment:
That's right, we want it to write a new file when we start this run through validation process and then append if we've already made changes to it in this run of certbot.
If it was always append and there was a stale file there I think it could cause a lot of problems